src/tarball.py

```diff
@@ -56,8 +56,20 @@ def _find_build_config(self, node):
 
     def _update_repo(self, config):
         self.log.info(f"Updating repo for {config.name}")
-        kernelci.build.update_repo(config, self._kdir)
+        try:
+            kernelci.build.update_repo(config, self._kdir)
+        except Exception as err:
+            self.log.error(f"Failed to update repo: {err}")
+            # safeguard, make sure it is git repo
+            if not os.path.exists(os.path.join(self._kdir, '.git')):
+                self.log.error(f"{self._kdir} is not a git repo")
+                return
```

Comment:
Please use the same return type throughout the function

Reply:
I think in such situation better even to raise exception